NebulaWorld/Player/DroneManager.cs

```diff
@@ -46,5 +56,20 @@ public static int[] GetPlayerDronePlans(ushort playerId)
             }
             return null;
         }
+
+        public static void AddBuildRequestSent(int entityId)
+        {
+            PendingBuildRequests.Add(entityId);
+        }
+
+        public static bool IsPendingBuildRequest(int entityId)
+        {
+            return PendingBuildRequests.Contains(entityId);
+        }
+
+        public static void RemoveBuildRequest(int entityId)
+        {
+            PendingBuildRequests.Remove(entityId);
```

Comment:
Allow notifying caller if the entry was removed.

```suggestion
        public static bool RemoveBuildRequest(int entityId)
        {
            return PendingBuildRequests.Remove(entityId);
```

Reply:
I am not sure if it would be helpful, maybe just to print a warning to logs that some exceptional case occured?
Since:

- Item is added to the array when the build request is sent to the host.
- item is removed from the array when the response from the host is received.

Reply:
added 👍